sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -22,6 +22,16 @@ select 5 / 0;
 select 5 / null;
 select null / 5;
 
+-- integral div
+select 5 div 2;
+select 5 div 0;
+select 5 div null;
+select null div 5;
+select cast(51 as decimal(10, 0)) div cast(2 as decimal(2, 0));
+select cast(5 as decimal(1, 0)) div cast(0 as decimal(2, 0));
+select cast(5 as decimal(1, 0)) div cast(null as decimal(2, 0));
+select cast(null as decimal(1, 0)) div cast(5 as decimal(2, 0));
```

Comment:
Were the four tests above newly added in this pr?

Reply:
They were already there. They were added when we support decimal in `IntegralDivide`